modules/contacts/hm-contacts.php

```diff
@@ -25,11 +25,24 @@ public function add_contact($data) {
         return true;
     }
 
-    public function get($id, $default=false) {
-        if (!array_key_exists($id, $this->contacts)) {
-            return $default;
+    public function get($id, $email_address="", $default=false) {
+        if(array_key_exists($id, $this->contacts)) {
```

Comment:
There are various spacing issues with the code, please run through a suitable editor or manually address. Try to be in sync with existing code - e.g. space after if/for/etc. operators, no space after function and before () brackets. See the diff here on github and you will notice all these places.

Reply:
Done

Reply:
@henochit now it is all good but the imap/site.js file - it went from 4 to 2 spaces for indentation and also some single quotes changed to double. Can you please reset the file to its original content and apply only the changes you want to apply and not such global formatting changes?